src/pages/BlockRegistration.tsx

```diff
@@ -67,6 +68,7 @@ function getRegistrationFormTissue(): RegistrationFormTissue {
 function buildRegistrationSchema(registrationInfo: GetRegistrationInfoQuery) {
   const validation = new RegistrationValidation(registrationInfo);
   return Yup.object().shape({
+    workNumbers: Yup.array().min(1, 'Atleast one work number must be selected'),
```

Comment:
`'At least one work number must be selected.'`

Reply:
Done